src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -157,45 +157,23 @@
 
 @CommandLineProgramProperties(
         summary =
-                "Checks that all data in the set of input files appear to come from the same " +
-                        "individual. Can be used to cross-check readgroups, libraries, samples, or files. " +
-                        "Operates on SAM/BAM/CRAM and VCF (including gVCF and gzipped-VCF). " +
+                "Checks the odds that all data in the set of input files come from the same individual. Can be used to cross-check readgroups, libraries, samples, or files. Acceptable inputs include BAM/SAM/CRAM and VCF/GVCF files. Output delivers LOD scores in the form of a CrosscheckMetric file. \n" +
                         "\n" +
                         "<h3>Summary</h3>\n" +
-                        "Checks if all the genetic data within a set of files appear to come from the same individual. " +
-                        "It quickly determines whether a group's genotype matches that of an input file by selective sampling, " +
-                        "and has been designed to work well for low-depth SAM (as well as high depth ones and VCFs.) " +
-                        "The tool collects fingerprints (essentially, genotype information from different parts of the genome) " +
-                        "at the finest level available in the data (readgroup for read-data files " +
-                        "and sample for variant-data files) and then optionally aggregates it by library, sample or file, to increase power and provide " +
-                        "results at the desired resolution. Output is in a \"Moltenized\" format, one row per comparison. The results are " +
-                        "emitted into a CrosscheckMetric metric file. " +
-                        "In this format the output will include the LOD score and also tumor-aware LOD score which can " +
-                        "help assess identity even in the presence of a severe loss of heterozygosity with high purity (which could cause it to " +
-                        "otherwise fail to notice that samples are from the same individual.) " +
-                        "A matrix output is also available to facilitate visual inspection of crosscheck results.\n " +
+                        "CrosscheckFingerprints rapidly checks the odds that all of the genetic data within a set of files come from the same individual. This is accomplished by selectively sampling from the input files, and determining whether the genotypes of the specified readgroups match to each other. \n " +
```

Comment:
'groups' was replaced with 'readgroups' which isn't always true. the "groups" are defined by the input and the argument `CROSSCHECK_BY`. And can be READ_GROUP, LIBRARY, SAMPLE, or FILE. That's why it was called "Group".